spanner/transaction.go

```diff
@@ -1890,7 +1890,7 @@ func (t *writeOnlyTransaction) applyAtLeastOnce(ctx context.Context, ms ...*Muta
 		for {
 			if sh == nil || sh.getID() == "" || sh.getClient() == nil {
 				// No usable session for doing the commit, take one from pool.
-				sh, err = t.sp.take(ctx)
+				sh, err = t.sp.takeMultiplexed(ctx)
```

Comment:
I assume that it falls back to regular sessions in case there is an issue with multiplexed sessions?

Reply:
Yes it will fallback to regular session, that logic is handled in `takeMultiplexed` because that behaviour is common